[WIP] New design pss examples

apozharski opened the pull request.

apozharski commented:
All non-benchmark examples have been updated. Do we want to continue to support the benchmark examples or perhaps move them to a separate directory.

nurkanovic commented:
> All non-benchmark examples have been updated. Do we want to continue to support the benchmark examples or perhaps move them to a separate directory.

We should put all benchmarks in a separate folder, but they should be adapted to the new syntax so that they can be run.

nurkanovic reviewed (changes requested):
I did some further polishing of the examples - mostly adding comments and removing outdated or unused lines of code. Only few issues are left and we can merge 
There are few PSS problems that are broken and/or use old design

- main_irma in the gene regulation
- ivp_example (I would rename it into initial_value_ocp)
- sliding mode ocp benchmark not in separate folder

apozharski commented:
>     * main_irma in the gene regulation
fixed in PR #124, as this is a `nosnoc.model.Heaviside` example not a `nosnoc.model.Pss` example

>     * ivp_example (I would rename it into initial_value_ocp)

Fixed in PR #124 

>     * sliding mode ocp benchmark not in separate folder

Benchmark folder moves have been done in the next PR #124.

apozharski commented on examples/a_simple_tutorial/pss_tutorial_turbo_car_ocp.m:
```diff
@@ -0,0 +1,101 @@
+solver_settings = MPECSolverOptions();
+solver_settings.relax_and_project_iters = 3;
+solver_settings.initalization_strategy = "RelaxAndProject";
+[ocp_solver,stats] = mpec_active_set_nosnoc(ocp_solver,solver_settings);
```
Hmmmmmmmmm :sweat_smile:
